### DISCUSSION TITLE
OpenEBS has been archived in CNCF?

### DISCUSSION THREAD

**Discussion by cmontemuino** [openebs/openebs#3777] [General] [answered]
I've noticed last week that OpenEBS shows in the Archived Project page from CNCF: https://www.cncf.io/archived-projects/

Is it the case that Mayastor is becoming a separate project and added to CNFC or something else happened?

**Comment by avishnu** [comment, marked as answer]
@cmontemuino A major restructuring of OpenEBS happened as part of the 4.0.0 major release, and a process for re-application to CNCF has been initiated.
Please check: https://github.com/cncf/sandbox/issues/104

**Reply by cmontemuino** [reply]
Thanks for the info. It's clear now.
